src/components/DataView/index.js

```diff
@@ -1,95 +1,206 @@
-import { h, Component } from "preact";
-import { doMatrixCheck } from "matrix-magic";
-import styled from "preact-emotion";
-import Matrix from "./Matrix";
-import Array from "./Array";
-import NArray from "./NArray";
-import { minDevice } from "../../lib/css";
+/* eslint-disable react/jsx-no-bind */
+import { Motion } from 'data-driven-motion';
+import { getMatrixDimensions } from 'matrix-magic';
+import styled from 'preact-emotion';
+import { makeCoordinateArray as makeMatrixArray } from './Matrix';
+import { makeCoordinateArray as makeArrayArray } from './Array';
+import { makeCoordinateArray as makeNArrayArray } from './NArray';
+import Cell from './Cell';
+import { minDevice } from '../../lib/css';
+import { max, min } from 'd3-array';
+import { scaleLinear } from '@vx/scale/build/index';
 
-const borderRadius = "10px";
+const STIFF_SPRING = { stiffness: 1028, damping: 36 };
```

Comment:
You can change these numbers.

http://chenglou.github.io/react-motion/demos/demo5-spring-parameters-chooser/